src/app/beer_garden/requests.py

```diff
@@ -1052,6 +1027,7 @@ def handle_event(event):
 
                     if getattr(existing_request, field) != new_value:
                         request_changed[field] = new_value
```

Comment:
Instead, you could use `request_changed` as a boolean value. Then when there is a change, set it to `True`